cloudinary-core/src/main/java/com/cloudinary/utils/Base64Map.java

```diff
@@ -0,0 +1,76 @@
+package com.cloudinary.utils;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Base64Map {
+    public static Map<String, String> values;
+
+    static {
+        values = new HashMap<>();
+        values.put("000000", "A");
```

Comment:
no better way to populate a hashmap in Java?

Reply:
No :(, Java 6, this was a nightmare
With Java 8 this could have been done much easier.